app/jobs/reports/sp_issuer_user_counts_report.rb

```diff
@@ -0,0 +1,30 @@
+require 'identity/hostdata'
+require 'json'
+
```

Comment:
these are usually required elsewhere, did things not work without them?
```suggestion
```

Reply:
I probably just copied them from the deleted_user_report...